### PR TITLE
Sync Serverless CI config/eventing-istio.yaml

### PR DESCRIPTION
**Pull request by pierDipi**
Add eventing-istio, ref: https://github.com/openshift-knative/hack/pull/35

### CODE REVIEW

**Comment by pierDipi** [comment]
I don't expect this test to run fine now, I will debug them on the actual branch

File: ci-operator/config/openshift-knative/eventing-istio/OWNERS [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Fetched from https://github.com/openshift-knative/eventing root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alanfx
+- aliok
+- creydr
+- jcrossley3
```
Comment (matzew) [inline ci-operator/config/openshift-knative/eventing-istio/OWNERS:11]: I think we can unfortunately remove Jim here

**Review by matzew — APPROVED** [review]
/lgtm
/approve

File: ci-operator/config/openshift-knative/eventing-istio/OWNERS (hunk 1 above)
Comment (pierDipi) [reply ci-operator/config/openshift-knative/eventing-istio/OWNERS:11]: I copied it from eventing directory

**Pull request merged**